bin/license.py

```diff
@@ -0,0 +1,381 @@
+    "javax.servlet.jsp:jsp-api:2.1": "(CDDL + GPLv2 with classpath exception) Java Servlet API (javax.servlet.jsp:jsp-api:2.1 - https://javaee.github.io/javaee-jsp-api)",
+    "javax.servlet:servlet-api:2.5": "(CDDL + GPLv2 with classpath exception) Java Servlet API (javax.servlet:servlet-api:2.5 - http://servlet-spec.java.net)",
+    "oro:oro:2.0.8": "(Apache License, Version 1.1) ORO (oro:oro:2.0.8 - https://mvnrepository.com/artifact/oro/oro)",
+    "org.hyperic:sigar:1.6.5.132": "(Apache License, Version 2.0) Sigar (org.hyperic:sigar:1.6.5.132 - https://github.com/hyperic/sigar)"
+}
+licenses_describe_map = {
+    "Apache 2.0 License":
+        """The following components are provided under the Apache License. See project link for details.
+The text of each license is the standard Apache 2.0 license.
+""",
+    "MIT License": """The following components are provided under the MIT License. See project link for details.
+The text of each license is also included at licenses/LICENSE-[project].txt.
+""",
+    "BSD License": """The following components are provided under a BSD license. See project link for details.
+The text of each license is also included at licenses/LICENSE-[project].txt.
+""",
+    "CC0-1.0 License": """The following components are provided under the CC0-1.0 License. See project link for details.
+The text of each license is also included at licenses/LICENSE-[project].txt.
+""",
+    "CDDL License": """The following components are provided under the CDDL License. See project link for details.
+The text of each license is also included at licenses/LICENSE-[project].txt.
+""",
+    "Eclipse Public License": """The following components are provided under the Eclipse Public License. See project link for details.
+The text of each license is also included at licenses/LICENSE-[project].txt.
+""",
+    "Public Domain License": """The following components are provided under the Public Domain License. See project link for details.
+The text of each license is also included at licenses/LICENSE-[project].txt.
+""",
+    "Mozilla Public License Version 2.0": """The following components are provided under the Mozilla Public License. See project link for details.
+The text of each license is also included at licenses/LICENSE-[project].txt.
+""",
+    "Unicode License": """The following components are provided under the Unicode License. See project link for details.
+The text of each license is also included at licenses/LICENSE-[project].txt.
+""",
+    "Go License": """The following components are provided under the Go License. See project link for details.
+The text of each license is also included at licenses/LICENSE-[project].txt.
+""",
+    "Other License": """The following components are provided under some minority Licenses. See project link for details.
+The text of each license is also included at licenses/LICENSE-[project].txt.
```

Comment:
This is not needed I think

Reply:
Hi @kezhenxu94, it is no need for now, but we still need collect unexpected license in case, right

Reply:
BTW, it would not be added to LICENSE file since all of unknown dependencies have been taken care.

Reply:
All future unexpected licenses should be taken care by human beings and categorize explicitly, I don't see there is any point to keep this, people may just run this script and commit the `Other License`, so if there is any unknown license, I'd rather just fail the command and abort, so that people can identify it manually

Reply:
@kezhenxu94 It's done, you can go check the latest version, thanks a lot.